### ISSUE TITLE
UnicodeError on log message

### ISSUE THREAD

**Issue by italomaia** [sebdah/scrapy-mongodb#26]
Just got this message in my deploy:
```
File "/home/myuser/.virtualenvs/stealth/local/lib/python2.7/site-packages/scrapy_mongodb.py", line 104, in __init__
	    self.config['collection']))
	UnicodeEncodeError: 'ascii' codec can't encode characters in position 27-28: ordinal not in range(128)
```
I believe the problem is related with passwords with non ascii characters.

**Comment by sebdah** [comment]
Will take a look on this, thanks for reporting.

**Comment by sebdah** [comment]
What characters do you have in the `collection` variable?

**Comment by italomaia** [comment]
collection characters: ['a', 'c', 'd', 'e', 'p', 'r', 's']
I believe the problem is with the password inside the uri. It has characters like 'ã' and 'ç'

**Comment by sebdah** [comment]
Thanks for the PR. I have merged it now and released version 0.8.2.
